config/autoload/doctrine.local.development.php.dist

```diff
@@ -40,16 +40,16 @@ return [
             'orm_default' => [
                 // metadata cache instance to use. The retrieved service name will
                 // be `doctrine.cache.$thisSetting`
-                'metadata_cache'    => 'array',
+                'metadata_cache'    => 'memcached',
```

Comment:
Correct me if I'm mistaken, but this setup doesn't seem to include a memcached server; so how does this exactly work?

Reply:
Also, why would you want to cache these things in development?

Reply:
Okay, for development it is not really relevant. I did install a memcached server in the docker container (see Dockerfile)

Reply:
I did however forgot to start the memcached server :p, but that's fixed now too.